great_expectations/data_context/data_context.py

```diff
@@ -1229,48 +1228,29 @@ def build_data_docs(self, site_names=None, resource_identifiers=None):
 
         return index_page_locator_infos
 
-    def clean_data_docs(self, site_name=None):
-         if not os.path.isdir(self.root_directory):
-            raise ge_exceptions.DataContextError(
-                "The data docs site and project root directory must be an existing directory to clean "
-            )
-
-         ge_dir = os.path.join(self.root_directory, site_name)
-         if not os.path.isdir(ge_dir):
-             raise ge_exceptions.DataContextError(
-                 "The data docs site and project root directory must be an existing directory to clean "
-             )
-         data_docs_urls = self.get_docs_sites_urls()
-         for site_dict in data_docs_urls:
-             myval = site_dict["site_url"][7:]
-             os.rename(myval,"/tmp/index.html")
-             return True
-         return False    
-         #module_name = 'great_expectations.render.renderer.site_builder'
-         """
-         module_name = 'great_expectations.data_context.store'
-         site_builder = SiteBuilder(
-            data_context=self,
-            runtime_environment={
-                "root_directory": self.root_directory
-            },
-            config_defaults={
-                            "module_name": module_name
-                        }
-         )
-         ldocs = context.get_docs_sites_urls(self)
-         return ldocs
-         """
-         #site_builder.clean_store_keys()
-         #return True
-         """
-         if safe_rrmdir(ge_dir, exist_ok=True):
-             clean_store_keys(site_builder)
-             return True
-         else:
-             return False
-         """
-         
+    def clean_data_docs(self, sname=None):
+        sites = self.get_docs_sites_urls()
+        print(sites)
```

Comment:
```suggestion
```

Reply:
Refresh. I tend to keep pushing not to lose work:)

Reply:
WIP, looks resolved